### PR TITLE
[cloud-providers-azure] vNetCIDR field description improved

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
vNetCIDR field description improved

## Why do we need it, and what problem does it solve?
https://github.com/deckhouse/deckhouse/issues/3944

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-providers-azure
type: chore
summary: Improve `vNetCIDR` field description.
impact_level: low 
```

### CODE REVIEW

File: candi/cloud-providers/azure/openapi/cluster_configuration.yaml [hunk 1]
```diff
@@ -82,6 +82,8 @@ apiVersions:
       subnetCIDR:
         description: |
           A subnet from the `vNetCIDR` address space for cluster nodes.
+
```
Comment (yalosev) [inline candi/cloud-providers/azure/openapi/cluster_configuration.yaml:85]: It's a subnetCIDR block. You need vNetCIDR

**Review by yalosev — DISMISSED** [review]

File: candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml [hunk 2]
```diff
@@ -38,10 +38,13 @@ apiVersions:
           По умолчанию `*`.
       vNetCIDR:
         description: |
-          Адресное пространство виртуальной сети в формате [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
+          Адресное пространство [виртуальной сети](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) в формате [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml:41]:
```suggestion
          Адресное пространство [виртуальной сети](https://learn.microsoft.com/ru-ru/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) в формате [CIDR](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D1%81%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%BE%D0%B2%D0%B0%D1%8F_%D0%B0%D0%B4%D1%80%D0%B5%D1%81%D0%B0%D1%86%D0%B8%D1%8F).
```

File: candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml [hunk 3]
```diff
@@ -38,10 +38,13 @@ apiVersions:
           По умолчанию `*`.
       vNetCIDR:
         description: |
-          Адресное пространство виртуальной сети в формате [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
+          Адресное пространство [виртуальной сети](https://learn.microsoft.com/ru-ru/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) в формате [CIDR](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D1%81%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%BE%D0%B2%D0%B0%D1%8F_%D0%B0%D0%B4%D1%80%D0%B5%D1%81%D0%B0%D1%86%D0%B8%D1%8F).
+          Виртуальная сеть — это виртуальная изолированная часть общедоступной сети Azure. Каждая виртуальная сеть выделена для вашей подписки.
+          **Важно!** Надо учитывать что в случаи настройки пиринга, vpn или связывания сетей других кластеров значение не должно пересекаться.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml:43]:
```suggestion
          Адресное пространство [виртуальной сети](https://learn.microsoft.com/ru-ru/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) в формате [CIDR](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D1%81%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%BE%D0%B2%D0%B0%D1%8F_%D0%B0%D0%B4%D1%80%D0%B5%D1%81%D0%B0%D1%86%D0%B8%D1%8F).

          Виртуальная сеть — это виртуальная изолированная часть общедоступной сети Azure. Каждая виртуальная сеть выделена для вашей подписки.
          
          **Важно!** Нужно учитывать, что в случае настройки пиринга, использования VPN или связывания сетей других кластеров, адресное пространство сетей не должно пересекаться.
```

File: candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml [hunk 4]
```diff
@@ -38,10 +38,13 @@ apiVersions:
...
       subnetCIDR:
         description: |
-          Подсеть из адресного пространства `vNetCIDR`, в которой будут работать узлы кластера.
+          [Подсеть](https://learn.microsoft.com/ru-ru/azure/virtual-network/virtual-network-vnet-plan-design-arm#subnets) из адресного пространства `vNetCIDR`, в которой будут работать узлы кластера.
+          Виртуальную сеть можно сегментировать на одну или несколько подсетей в пределах ограничений.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml:47]:
```suggestion
          [Подсеть](https://learn.microsoft.com/ru-ru/azure/virtual-network/virtual-network-vnet-plan-design-arm#subnets) из адресного пространства `vNetCIDR`, в которой будут работать узлы кластера.

          Виртуальную сеть можно сегментировать на одну или несколько подсетей в пределах ограничений.
```

File: candi/cloud-providers/azure/openapi/cluster_configuration.yaml [hunk 5]
```diff
@@ -75,13 +75,16 @@ apiVersions:
           By default, `*`.
       vNetCIDR:
         description: |
-          An address space of the virtual network in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) format.
+          An address space of the [virtual network](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) format.
+          A virtual network is a virtual, isolated portion of the Azure public network. Each virtual network is dedicated to your subscription.
+          **Caution!** Please note that if you are setting up peering, vpn or linking networks of other clusters, the value should not overlap.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/cluster_configuration.yaml:80]:
```suggestion
          An address space of the [virtual network](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) format.

          A virtual network is a virtual, isolated portion of the Azure public network. Each virtual network is dedicated to your subscription.

          **Note!** If you are setting up peering, using vpn or linking networks of other clusters, network address spaces should not overlap.
```

File: candi/cloud-providers/azure/openapi/cluster_configuration.yaml [hunk 6]
```diff
@@ -75,13 +75,16 @@ apiVersions:
...
         type: string
         pattern: '^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$'
         example: 10.0.0.0/16
       subnetCIDR:
         description: |
-          A subnet from the `vNetCIDR` address space for cluster nodes.
+          A [subnet](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#subnets) from the `vNetCIDR` address space for cluster nodes.
+          A virtual network can be segmented into one or more subnets up to the limits.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/cluster_configuration.yaml:87]:
```suggestion
          A [subnet](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#subnets) from the `vNetCIDR` address space for cluster nodes.

          A virtual network can be segmented into one or more subnets up to the limits.
```

File: candi/cloud-providers/azure/openapi/cluster_configuration.yaml [hunk 7]
```diff
@@ -75,13 +75,19 @@ apiVersions:
           By default, `*`.
       vNetCIDR:
         description: |
-          An address space of the virtual network in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) format.
+          An address space of the [virtual network](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#virtual-networks) in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) format.
+
+          A virtual network is a virtual, isolated portion of the Azure public network. Each virtual network is dedicated to your subscription.
+
+          **Note!** If you are setting up peering, using vpn or linking networks of other clusters, network address spaces should not overlap.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/cluster_configuration.yaml:82]:
```suggestion
          **Caution!** If you are setting up peering, using vpn or linking networks of other clusters, network address spaces should not overlap.
```

**Pull request merged**